convert_batch_prediction_output_to_submission.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility to convert the output of batch prediction into a CSV submission."""
+
+import json
+import tensorflow as tf
+
+from builtins import range
+from tensorflow import app
+from tensorflow import flags
+from tensorflow import gfile
+from tensorflow import logging
+
+
+FLAGS = flags.FLAGS
+
+if __name__ == '__main__':
+
+  flags.DEFINE_string(
+      "prediction_files_pattern", None,
+      "Pattern specifying the list of files that contain the output of batch "
+      "prediction.")
+  flags.DEFINE_string(
+      "output_file", None,
+      "The file to save the predictions to.")
```

Comment:
I changed the name of the script to convert_prediction_from_json_to_csv.py I changed the name of the flags to json_prediction_files_pattern and csv_output_file. I added more details in the head description of the script, and in the flag descriptions.